Editor/Scripts/BotManagement/RGBotSynchronizer.cs

```diff
@@ -76,14 +368,32 @@ private string CreateBotFolder(string botName, long botId)
 
         private void CreateNewBotAssets(string folderName, string botName, long botId)
```

Comment:
Should we call our synchronize call immediately after creating a new bot?

Reply:
I wouldn't personally.  If we did, I would only consider pushing that 1 bot, not pulling down or updating all the others as that could take quite a bit of time and bandwidth depending on your number of bots.

Reply:
For context; I say I wouldn't because getting the id reserved doesn't do a whole lot and you won't have your bot code in place right after creating it from the menu.